recipe/open_math_reasoning/run_sft_qwen3_8b.sh

```diff
@@ -55,7 +55,7 @@ MEGATRON_ENGINE_CONFIG="\
     engine.pipeline_model_parallel_size=${PP_SIZE} \
     engine.virtual_pipeline_model_parallel_size=${VPP_SIZE} \
     engine.context_parallel_size=${CP_SIZE} \
-    engine.use_mbridge=False"
+    engine.use_mbridge=True"
```

Comment:
We should also set `use_mbridge` to True? https://github.com/volcengine/verl/blob/main/verl/trainer/config/engine/megatron.yaml#L77-L78